src/Simplex/Messaging/Agent.hs

```diff
@@ -127,7 +130,12 @@ getSMPAgentClient cfg initServers = newSMPAgentEnv cfg >>= runReaderT runAgent
     runAgent = do
       c <- getAgentClient initServers
       void $ race_ (subscriber c) (runNtfSupervisor c) `forkFinally` const (disconnectAgentClient c)
+      restoreCommands c
       pure c
+    restoreCommands c =
+      runExceptT (withStore' c $ \db -> getPendingCommandsServers db) >>= \case
+        Left e -> liftIO $ print e
+        Right servers -> forM_ servers $ \s -> runExceptT (resumeCommandProcessing c s)
```

Comment:
this whole thing needs to be in AgentMonad, to avoid multiple runExceptT

Reply:
what's more important, I think it should only restore connection-specific commands on connection subscription, so there should be nothing to restore on agent start.